tests/terraform/graph/graph_builder/test_local_graph.py

```diff
@@ -322,3 +322,22 @@ def test_variables_same_name_different_modules(self):
         # Check they point to 2 different modules
         self.assertEqual(2, len(module_variable_edges))
         self.assertNotEqual(local_graph.vertices[module_variable_edges[0].origin], local_graph.vertices[module_variable_edges[1].origin])
+
+    def test_nested_modules_address_attribute(self):
```

Comment:
can you move the test to `tests/terraform/graph/graph_builder/test_graph_builder.py` and use the `get_vertex_by_name_and_type` function to get the vertices? 
this way the test won't break if we add the missing vertices to the graph and it will be clear which vertices we are checking.